src/react/components/pages/projectSettings/projectSettingsPage.tsx

```diff
@@ -87,6 +88,8 @@ export default class ProjectSettingsPage extends React.Component<IProjectSetting
         await this.ensureSecurityToken(project);
         await this.props.projectActions.saveProject(project);
 
+        toast.success(interpolate(strings.projectSettings.messages.saveSuccess, { project }));
```

Comment:
@tbarlow12 - This is a way we can handle string interpolation of messages even if the ${token} is in the middle of a localized message.

Reply:
This is awesome